tests/coalaCITest.py

```diff
@@ -59,6 +59,7 @@ def test_show_patch(self):
                 "-f", re.escape(filename),
                 "-b", "SpaceConsistencyTestBear",
                 "--settings", "use_spaces=True")
+            self.assertIn("Line contains ", output)  # Result message is shown
```

Comment:
This will pass regardless if the result is shown on top or not, right?
I wonder if there is a way to test that it will be on top. If there is, we should test for test.
As an idea, you can create a regexp and check for something like `<string that should be first>.*<string that should be second>.*<and so on..>`. Or, strip stuff off as you check for inclusions. If it is in separate lines, maybe you can split output linewise and check for sub-sequence